pkg/vault/approle.go

```diff
@@ -19,7 +21,11 @@ func (a *AppRole) Login() error {
 		return err
 	}
 
-	SetToken(a.Client, data.Auth.ClientToken)
+	err = SetToken(a.Client, data.Auth.ClientToken)
+	if err != nil {
+		fmt.Printf("Could not write token to file: %s", err.Error())
```

Comment:
Can we add a message like "- will need to login to Vault on subsequent runs" ?

Reply:
Sure